build.sh

```diff
@@ -81,6 +81,12 @@ for BINARY in $BINARIES; do
     error "Unable to find output"
     exit 1
   fi
+  
+  if [ "$MINIFY" = "true" ]; then
```

Comment:
Add default value for MINIFY here `${MINIFY:-""}` to prevent crash if it is not set

Reply:
@Douile, I added the default value as you said but if the variable is not set, the condition will simply evaluated to false
Let me know if you want another change